### ISSUE TITLE
bug: aspect ratio of profile photo when clicking top right profile image button

### ISSUE THREAD

**Issue by identor** [trenchdevs/trenchdevs#7]
![image](https://user-images.githubusercontent.com/3519612/92074494-cade8e00-ede8-11ea-9801-140589e64dba.png)

See image ^

Notice profile image in callout / popup is a little bit distorted.

**Comment by christopheredrian** [comment]
Understood! 
Maybe its better to store these thumbnails after uploading the original avatar

**Comment by identor** [comment]
Agreed. For sizes I suggest we use consistent aspect ratio, say 1:1. For thumbnails saw that it's use in two places, size suggestions:

1. `portfolio/edit` page - 400x400
2. top right dashboard image - 40x40
